apps/user/components/BottomNavigation.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { cn } from "@ui/lib/utils";
+import { Bell, Calendar, UserRound } from "lucide-react";
+import React, { useState } from "react";
+
+type NavigationItemsProp = {
+  label: string;
+  path: string;
+  icon: React.ElementType;
+};
+
+const NAVIGATION_ITEMS: NavigationItemsProp[] = [
+  { label: "캘린더", path: "", icon: Calendar },
+  { label: "알림", path: "alarm", icon: Bell },
+  { label: "마이페이지", path: "my-page", icon: UserRound },
+];
+
+export default function BottomNavigation() {
+  const [page, setPage] = useState("");
+
+  const handleClickChangePage = (path: string) => {
+    setPage(path);
+  };
```

Comment:
상태를 변경하는 것이 아니라 실제 url을 변경하는 로직이어야 합니다. 아직 page 개발 단계가 아니기 때문에 TODO 주석처리만 해주셔도 될 것 같습니다
```suggestion
  const handleClickChangePage = (path: string) => {
    //TODO: url 이동 로직 추가
  };
```

Reply:
스토리북 동작을 위해 추가해놓았는데 수정하겠습니다!